PrintRefManual.cpp

```diff
@@ -0,0 +1,24 @@
+#include "PrintRefManual.h"
+
+namespace TelCoColorCoder {
+	void printRefManual()
+	{
+		std::cout << "Color Pair to Pair Number Reference Manual" << std::endl;
+		std::cout << std::endl;
+		std::cout << std::endl;
+
+		int pairNo = 0;
+
+		for (int majorCol = 0; majorCol < numberOfMajorColors; majorCol++) {
+			for (int minorCol = 0; minorCol < numberOfMinorColors; minorCol++) {
+				pairNo = GetPairNumberFromColor(static_cast<MajorColor>(majorCol), static_cast<MinorColor>(minorCol));
+
+				std::cout << "Major Color : " << MajorColorNames[majorCol] << ", " << "Minor Color : " 

```

Comment:
Consider separating all columns with commas (instead of `=` before the `pairNo`)
Then the output can be a csv file - Excel will take care of formatting it for us.
Use off-the-shelf functionality whenever possible.

Reply:
Noted. I will keep this in mind. For now I have changed the Printing code.